<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick Data Logger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="swPages">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="swpStart">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Quick Data Logger&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Working mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="rbLptMode">
            <property name="text">
             <string>LPT mode (Onboard or PCI LPT port requared)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="rbFtdiMode">
            <property name="text">
             <string>FTDI mode (FTDI 2232/4232 USB connection requared)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbStartNext">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="swpFtdi">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;FTDI mode&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>State:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lDriverState_Ftdi">
            <property name="text">
             <string>FTDI Driver Error!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Recording channals:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="cbEndpointList_Ftdi">
            <item>
             <property name="text">
              <string>FTAAAAAA</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FTAAAAAB</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FTAAAAAC</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FTAAAAAD</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Speed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbSpeed_Ftdi">
            <property name="currentIndex">
             <number>7</number>
            </property>
            <item>
             <property name="text">
              <string>4.8 kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9.6 kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19.2 kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38.4 kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>76.8 kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.536 MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2.304 MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3.072 MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6.144 MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9.216 MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18.432 MHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36.864 Mhz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>73.728 Mhz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>147.456 Mhz</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Buffer count:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbBufferSize_Ftdi">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>64 k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256 k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>64 M</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Save as:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QCheckBox" name="cbRawSave_Ftdi">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>DAT file</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbOSDsave_Ftdi">
            <property name="text">
             <string>OSD file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbVCDsave_Ftdi">
            <property name="text">
             <string>VCD file</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Save to:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLineEdit" name="lePathToSave_Ftdi"/>
          </item>
          <item>
           <widget class="QToolButton" name="pbBrowse_Ftdi">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_Ftdi">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="pbCancel_Ftdi">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbRun_Ftdi">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="swpLpt">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;LPT mode&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>State:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lDriverState_Lpt">
            <property name="text">
             <string>WinIO Load Error!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>LPT port name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbPortName_Lpt">
            <item>
             <property name="text">
              <string>LPT1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LPT2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LPT3</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbMicronasMode_LPT">
            <property name="text">
             <string>Micronas mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Speed fixed at 400 kHz</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Buffer count (16 bit words):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbBufferSize_Lpt">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <item>
             <property name="text">
              <string>64 k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256 k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32 M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>64 M</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Save as:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QCheckBox" name="cbRawSave_Lpt">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>DAT file</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbOSDsave_Lpt">
            <property name="text">
             <string>OSD file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbVCDsave_Lpt">
            <property name="text">
             <string>VCD file</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Save to:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="lePathToSave_Lpt"/>
          </item>
          <item>
           <widget class="QToolButton" name="pbBrowse_Lpt">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_Lpt">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pbCancel_Lpt">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbRun_Lpt">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
